Authenticator/Source/OTPTokenListViewController.swift

```diff
@@ -214,17 +209,39 @@ extension OTPTokenListViewController /* UITableViewDelegate */ {
     }
 
     override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        let keychainItem = self.tokenManager.keychainItemAtIndex(indexPath.row)
-        if self.editing {
+        if let cell = tableView.cellForRowAtIndexPath(indexPath) as? TokenRowCell,
```

Comment:
Length Violation: Line should be 100 characters or less: currently 101 characters